radio/src/gui/128x64/model_setup.cpp

```diff
@@ -1074,15 +1074,34 @@ void menuModelSetup(event_t event)
             switch (menuHorizontalPosition) {
               case 0:
 #if defined(HARDWARE_INTERNAL_MODULE)
-                if (moduleIdx == INTERNAL_MODULE) {
+                if (moduleIdx == INTERNAL_MODULE) 
+                {
                   uint8_t moduleType = checkIncDec(event, g_model.moduleData[moduleIdx].type, MODULE_TYPE_NONE, MODULE_TYPE_MAX, EE_MODEL, isInternalModuleAvailable);
-                  if (checkIncDec_Ret) {
+                  #if defined(RADIO_Commando8)
+                    if (reusableBuffer.moduleSetup.newType!=MODULE_TYPE_NONE)
+                    {
+                      reusableBuffer.moduleSetup.newType=MODULE_TYPE_NONE;
+                      g_model.moduleData[moduleIdx].type = reusableBuffer.moduleSetup.newType;
+                      reusableBuffer.moduleSetup.previousType = reusableBuffer.moduleSetup.newType;
+                      setModuleType(EXTERNAL_MODULE, MODULE_TYPE_NONE);
+                    }
+                  #endif
```

Comment:
The built-in and external RF module cannot be turned on at the same time, because it controls the RF electronic switch that is connected to a Dual-Band antenna with 2.400 and 900 Mhz. Both RF modules use the same antenna. We tried to use the isInternalModuleAvailable() function and finally failed. Is there a better approach for our case?

Reply:
code fixed, please review